database/factories/sessionFactory.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use App\Attendance;
+use App\Event;
+use App\Session;
+use App\User;
+use Illuminate\Support\Str;
+use Faker\Generator as Faker;
+
+/*
+|--------------------------------------------------------------------------
+| Model Factories
+|--------------------------------------------------------------------------
+|
+| This directory should contain each of the model factory definitions for
+| your application. Factories provide a convenient way to generate new
+| model instances for testing / seeding your application's database.
+|
+*/
+
+$factory->define(Session::class, function (Faker $faker) {
+    // Ensuring that the ending dates are always a reasonable time after the starting dates.
+    $startingTime = $faker->optional($weight = 0.8)->dateTimeThisYear('+1 month');
+    if ($startingTime) {
```

Comment:
Should be consistent with previous variable names, 'startTime' rather than 'startingTime'